src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts

```diff
@@ -1,39 +1,32 @@
-import { Component, OnInit, OnChanges, OnDestroy, SimpleChanges, SimpleChange, Input, Output, Inject, EventEmitter } from '@angular/core';
-import { Router } from '@angular/router';
+import { Component, Inject, Input, OnChanges, OnDestroy, OnInit, SimpleChanges } from '@angular/core';
 
-import { IWorkflowStep, IWorkflowTransition, IWorkflow, WorkflowTransitionDirection } from '../../models/workflow';
-import { LoggerFactory, ILoggerDelegate } from '../../common/logger';
-import { INotifyPropertyChanged } from '../../core/component'
+import { IMultiSelectSettings, IMultiSelectTexts } from 'angular-2-dropdown-multiselect';
```

Comment:
I believe there are spaces at the end inside the {}. Are you thinking of something else ??